hadoop-hdds/server-scm/src/test/java/org/apache/hadoop/hdds/scm/block/TestDeletedBlockLog.java

```diff
@@ -300,7 +347,8 @@ public void testRandomOperateTransactions() throws Exception {
         // verify the number of added and committed.
         try (TableIterator<Long,
             ? extends Table.KeyValue<Long, DeletedBlocksTransaction>> iter =
-            scm.getScmMetadataStore().getDeletedBlocksTXTable().iterator()) {
+                 scm.getScmMetadataStore().getDeletedBlocksTXTable()
```

Comment:
Checkstyle complains this line exceeds 80 char limitation so I made this change.